local-macros/src/lib.rs

```diff
@@ -658,9 +658,8 @@ fn get_type_struct_interface_impl(
                     fn get_or_initialize_fresh_type(
                         &self,
                         type_checker: &crate::TypeChecker,
-                        wrapper: &::std::rc::Rc<crate::Type>,
```

Comment:
This explicit `wrapper` argument to `get_or_initialize_fresh_type()` is not longer necessary because we can call `self.type_wrapper()` to get the same thing